lnbits/extensions/withdraw/static/js/index.js

```diff
@@ -132,6 +137,9 @@ new Vue({
         id: this.formDialog.data.wallet
       })
       var data = _.omit(this.formDialog.data, 'wallet')
+      if (data.use_custom && !data?.custom_url) {
+        data.custom_url = '/static/images/default_voucher.png'
```

Comment:
Code style: maybe put the hard coded string into a constant (if that makes sense for JS).